bin/templates/scripts/cordova/run.bat

```diff
@@ -15,11 +15,5 @@
 :: specific language governing permissions and limitations
 :: under the License
 @ECHO OFF
-SET script_path="%~dp0run"
-IF EXIST %script_path% (
-        node %script_path% %*
-) ELSE (
-    ECHO.
-    ECHO ERROR: Could not find 'run' script in 'cordova' folder, aborting...>&2
-    EXIT /B 1
-)
+ECHO ERROR: Running 'ios' platform on Windows is not supported>&2
+EXIT /B 0
```

Comment:
It allows to build&run supported platforms without interruption in case we have several platforms added ([Jira issue with example](https://issues.apache.org/jira/browse/CB-9512)).